### PR TITLE
code refine for class CalanderSlotsHandler

### PR DESCRIPTION
**Pull request by Chen-Ben-yashar**
split long functions, magic numbers, code readability.

short and obvious strings **did not** become a magic number, i think it is over-doing it.
if you think otherwise, leave comment where you think is necessary.

### CODE REVIEW

File: app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java [hunk 1]
```diff
@@ -51,35 +57,40 @@
         this.topSelectionToDisplay = 3;
         this.context = context;
         this.view = view;
-        calendarRefForUpdate = db.collection("calendars").document(group.getGroupId());
```
Comment (orenpinto) [inline app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java]: cancel this change please

File: app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java [hunk 2]
```diff
@@ -196,7 +214,7 @@ public TimeSlot getSlotById(int slotId) {
     }
 
     private void readCalendarFromDB() {
-        calendarRef = db.collection("calendars").document(group.getGroupId());
+        calendarRef = db.collection(CALENDER_COLLECTION_PATH).document(group.getGroupId());
```
Comment (orenpinto) [inline app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java]: also here

File: app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java (hunk 1 above)
Comment (Chen-Ben-yashar) [reply app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java]: fixed

File: app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java (hunk 2 above)
Comment (Chen-Ben-yashar) [reply app/src/main/java/com/example/meetapp/CalendarSlotsHandler.java]: fixed

**Pull request closed without merging**